clearml/docker-compose-dev.yaml

```diff
@@ -0,0 +1,211 @@
+version: "3.6"
+services:
+
+  apiserver:
+    command:
+    - apiserver
+    container_name: clearml-apiserver
+    image: allegroai/clearml:latest
+    restart: unless-stopped
+    volumes:
+    - clearml-logs:/var/log/clearml
+    - clearml-config:/opt/clearml/config
+    - clearml-data-fileserver:/mnt/fileserver
+    depends_on:
+      - redis
+      - mongo
+      - elasticsearch
+      - fileserver
+    environment:
+      CLEARML_ELASTIC_SERVICE_HOST: elasticsearch
+      CLEARML_ELASTIC_SERVICE_PORT: 9200
+      CLEARML_ELASTIC_SERVICE_PASSWORD: ${ELASTIC_PASSWORD}
```

Comment:
Maybe move it to secrets? If it's acceptable to keep here, then okay

Reply:
We're using docker-compose for dev installations only where we don't use any secrets manager. For k8s installation these variables must be populated by secrets.
However, exact this docker-compose-dev.yaml doesn't specify how secretes are populated. I assume easiest way to do it - create `.env` file inside `clearml` directory